src/client/components/pages/TestsHistory/TestsHistory.jsx

```diff
@@ -78,20 +93,14 @@ const ExamsHistory = (props) => {
                 id: 1,
               },
               {
-                header: 'Categoría',
-                accessor: 'type',
+                header: 'Doctor que receta',
```

Comment:
**Assigned by doctor**
in spanish:
**Asignado por el doctor:**
dado que es un exámen no un jarabe

Reply:
Hahahaa ok. I'll change that too.

Reply:
Done :rocket: